classes/Referrer.php

```diff
@@ -359,7 +359,7 @@ public static function refreshIndex($referrers = null)
      *
      * @param int $idReferrer Referrer ID
      * @param int $idProduct Product ID
-     * @param int $idEmployee Employee ID
+     * @param Employee|null $idEmployee Employee ID
```

Comment:
weird that this variable is called `$idEmployee` when it's actually an `Employee` object, would it be a BC to rename the parameter ?

Reply:
I disagree. The scope of this PR is validate PHPStan with a minimal of BC Breaks.